packages/commonwealth/client/scripts/views/pages/view_thread/index.tsx

```diff
@@ -538,7 +538,7 @@ const ViewThreadPage = ({ identifier }: ViewThreadPageProps) => {
               label: thread.readOnly ? 'Unlock thread' : 'Lock thread',
               iconLeft: 'lock' as const,
               onClick: (e) => {
-                e.preventDefault();
+                // e.preventDefault();
```

Comment:
why commenting out the code? remove it. Also `e` is not used in this case so remove it. Same in example above

Reply:
![image](https://user-images.githubusercontent.com/14819225/223740520-a04e3497-a79d-4f25-a94a-3dbb56a399bc.png)
works in thread view but dose not work in 'thread preview menu'

Reply:
Addressed